src/templates/keyboardShortcut.ts

```diff
@@ -8,18 +8,15 @@ export interface KeyboardShortcut {
   hasMouse?: boolean;
 }
 
+export const shiftSymbol = "⇧" as const;
+
 /**
  * @description
  * A standardized template to display keyboard shortcuts and shortcut
  * combinations
  */
 export function keyboardShortcutTemplate(shortcut: KeyboardShortcut): HTMLTemplateResult {
-  const isAscii = shortcut.keys.every((key) => key.charCodeAt(0) < 128);
-  const hasUpperCase = shortcut.keys.some((key: string) => key === key.toUpperCase());
-  const shouldShowShift = hasUpperCase && isAscii;
-
   return html`
-    ${when(shouldShowShift, () => html`<kbd class="shortcut-legend">Shift</kbd> +`)}
```

Comment:
i think you misunderstood what i was saying. 

talk to with about this in the morning - i don't understand it